public/globals.js

```diff
@@ -855,7 +856,19 @@ window.pkp = {
 		'user.affiliations.primaryLocaleRequired':
 			'The primary language {$primaryLocale} is required',
 		'workflow.uploadRevisions': 'Upload revisions',
-			'userInvitation.searchField':'Search for a user by email address, username, or ORCID iD. Enter only one to get started!'
+		'userInvitation.searchField':
+			'Search for a user by email address, username, or ORCID iD. Enter only one to get started!',
+		'grid.user.enable': 'Enable User',
+		'grid.user.disable': 'Disable User',
+		'grid.user.remove': 'Remove User',
+		'grid.user.logInAs': 'Login As',
+		'grid.user.grid.user.enableReasonDescription':
+			"Once the user enabled, they will regain access to OJS, and you'll be able to invite them to roles as needed.",
```

Comment:
Looks like "is" is missing here:
```suggestion
			"Once the user is enabled, they will regain access to OJS, and you'll be able to invite them to roles as needed.",
```